package.json

```diff
@@ -15,7 +15,8 @@
     "format-fix": "prettier --write .",
     "fix": "npm run lint-fix && npm run format-fix",
     "ngrok": "ngrok http 8787",
-    "register": "ts-node-esm src/register.ts"
+    "register": "ts-node-esm src/register.ts",
+    "env-var-check": "ts-node-esm config/envVarCheck.ts"
```

Comment:
So this is for?
And where and when to use/run it?

Reply:
this env-var-check will run `config/envVarCheck.ts` in the first job in the Github Actions, [Environment-Variables-Check](https://github.com/Saitharun279/discord-slash-commands/blob/e29109b7fd5270e12741bd48b93cc2f6f8833c5e/.github/workflows/register-commands-staging.yaml#L6)